metricflow/cli/main.py

```diff
@@ -445,26 +447,27 @@ def metrics(cfg: CLIContext, show_all_dims: bool = False, search: Optional[str]
 
 @list.command()
 @click.option(
-    "--metric-names",
-    type=click_custom.SequenceParamType(),
-    help="List dimensions by given metrics (intersection). Ex. --metric-names bookings,messages",
+    "--metrics",
+    type=click_custom.SequenceParamType(min_length=1),
+    default="",
+    help="List dimensions by given metrics (intersection). Ex. --metrics bookings,messages",
 )
 @pass_config
 @exception_handler
 @log_call(module_name=__name__, telemetry_reporter=_telemetry_reporter)
-def dimensions(cfg: CLIContext, metric_names: List[str]) -> None:
+def dimensions(cfg: CLIContext, metrics: List[str]) -> None:
     """List all unique dimensions."""
     spinner = Halo(
         text="🔍 Looking for all available dimensions...",
         spinner="dots",
     )
     spinner.start()
 
-    dimensions = cfg.mf.simple_dimensions_for_metrics(metric_names)
+    dimensions = cfg.mf.simple_dimensions_for_metrics(metrics)
```

Comment:
Checking that entities aren't listed in the call?

Reply:
yup, we have this in the MFE
```
without_any_property=frozenset(
                    {
                        LinkableElementProperties.ENTITY,
                        LinkableElementProperties.DERIVED_TIME_GRANULARITY,
                        LinkableElementProperties.LOCAL_LINKED,
                    }
                ),
```
On a similar note, in a followup PR, @nhandel mentioned wanting to add entities being shown in the CLI (ie., list-entities, showing entities in list-metrics).